s3/src/main/scala/monix/connect/s3/S3.scala

```diff
@@ -885,6 +885,110 @@ trait S3 { self =>
     } yield s3Object
   }
 
+  /**
+    * Returns the most recently uploaded object in a bucket.
+    *
+    * ==Example==
+    *
+    * {{{
+    * import cats.effect.Resource
+    * import monix.connect.s3.S3
+    * import monix.eval.Task
+    * import software.amazon.awssdk.auth.credentials.{AwsBasicCredentials, StaticCredentialsProvider}
+    * import software.amazon.awssdk.regions.Region
+    * import monix.execution.Scheduler.Implicits.global
+    *
+    * val bucket = "my-bucket"
+    * val prefix = "prefix/to/list/keys"
+    * val s3AccessKey: String = "TESTKEY"
+    * val s3SecretKey: String = "TESTSECRET"
+    * val basicAWSCredentials = AwsBasicCredentials.create(s3AccessKey, s3SecretKey)
+    * val staticCredProvider = StaticCredentialsProvider.create(basicAWSCredentials)
+    *
+    * val s3: Resource[Task, S3] = S3.create(staticCredProvider, Region.AWS_GLOBAL)
+    *
+    * s3.use{s3 => s3.listLatestObject(bucket, prefix = Some(prefix)).toListL}.runToFuture
+    * }}}
+    *
+    * To use this operation in an AWS (IAM) policy, you must have permissions to perform
```

Comment:
Maybe add `@note` tag?

Reply:
@note tag? Is there an example you can point me too?

Reply:
disregard that